homeassistant/components/xiaomi_aqara.py

```diff
@@ -285,6 +287,9 @@ def push_data(self, data, raw_data):
         _LOGGER.debug("PUSH >> %s: %s", self, data)
         was_unavailable = self._async_track_unavailable()
         is_data = self.parse_data(data, raw_data)
+        print("=========================================================")
+        print(raw_data)
+        print("=========================================================")
```

Comment:
Remove all modifications and references to xiaomi_aqara. xiaomi_aqara implements the developer api access, xiaomi_miio is the correct one for this.